recipe/build.sh

```diff
@@ -0,0 +1,10 @@
+set -x
+export CPPFLAGS="${CPP_FLAGS} -I$PREFIX/include "
```

Comment:
```suggestion
export CPPFLAGS="${CPPFLAGS} -isystem $PREFIX/include "
```

Reply:
I wonder if this is actually needed now.